onyo/lib/commands.py

```diff
@@ -537,13 +537,14 @@ def onyo_mv(inventory: Inventory,
         subject = "mv"
         for s in sources:
             move_asset_or_dir(inventory, s, destination)
-    else:
-        # RENAME
+    elif len(sources) == 1 and inventory.repo.is_inventory_dir(sources[0]) and \
+            inventory.repo.is_inventory_dir(Path(*destination.parts[:-1])):
```

Comment:
Two things:

1. `Path(*destination.parts[:-1])` is a little strange. What you mean is `destination.parent`.

2. I don't think this last condition is needed. The first two make sure it's a directory rename already. `rename_directory` should fail on the destination criteria correctly, raising `InvalidInventoryOperation(f"Cannot rename {src} -> {dst}. Consider moving instead.")`. Which is more specific and accurate, than failing with the broader exception you raise from within here.

Reply:
1. Yes definetely right!
2. That is right from perspective of `rename_directory()`, but does in the other cases not adjust the issue this PR tries to solve. But I agree that the check could be simpler and `rename_directory()` addresses some issues more specifically, so `.is_dir()` checks here are sufficient.

Reply:
What other case? The two issues are both about targeting a non-existing dir. Whether `destination.parent` exists or not does not distinguish these two somehow. The difference between both issues is the source. That's the second condition - the third seems irrelevant. What's the case where the fact that `destination.parent` exists makes a difference?

Reply:
The difference when `destination.parent` exists is that it might be possibly a valid rename. Without that check comes the message `Consider moving instead.` which is not more specific, but just wrong -> you can't consider moving, because it is also not allowed.
In 100% of cases when the `destination.parent` does not exist, we know beforehand that a renaming will fail.

Reply:
Ok, considering that #426 seems likely to change more of that, I'll resolve here and merge. Let's only reconsider when looking at the whole picture.